hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleDispatcher.java

```diff
@@ -105,69 +118,56 @@ public PreHandleDispatcher(
-            case CRYPTOUPDATEACCOUNT -> cryptoHandler.preHandleUpdateAccount(transactionBody);
-            case CRYPTOTRANSFER -> cryptoHandler.preHandleCryptoTransfer(transactionBody);
-            case CRYPTODELETE -> cryptoHandler.preHandleCryptoDelete(transactionBody);
-            case CRYPTOAPPROVEALLOWANCE -> cryptoHandler.preHandleApproveAllowances(
-                    transactionBody);
-            case CRYPTODELETEALLOWANCE -> cryptoHandler.preHandleDeleteAllowances(transactionBody);
-            case CRYPTOADDLIVEHASH -> cryptoHandler.preHandleAddLiveHash(transactionBody);
-            case CRYPTODELETELIVEHASH -> cryptoHandler.preHandleDeleteLiveHash(transactionBody);
-
-            case FILECREATE -> fileHandler.preHandleCreateFile(transactionBody);
-            case FILEUPDATE -> fileHandler.preHandleUpdateFile(transactionBody);
-            case FILEDELETE -> fileHandler.preHandleDeleteFile(transactionBody);
-            case FILEAPPEND -> fileHandler.preHandleAppendContent(transactionBody);
-
-            case FREEZE -> freezeHandler.preHandleFreeze(transactionBody);
-
-            case UNCHECKEDSUBMIT -> networkHandler.preHandleUncheckedSubmit(transactionBody);
-
-            case SCHEDULECREATE -> scheduleHandler.preHandleCreateSchedule(transactionBody);
-            case SCHEDULESIGN -> scheduleHandler.preHandleSignSchedule(transactionBody);
-            case SCHEDULEDELETE -> scheduleHandler.preHandleDeleteSchedule(transactionBody);
-
-            case TOKENCREATION -> tokenHandler.preHandleCreateToken(transactionBody);
-            case TOKENUPDATE -> tokenHandler.preHandleUpdateToken(transactionBody);
-            case TOKENMINT -> tokenHandler.preHandleMintToken(transactionBody);
-            case TOKENBURN -> tokenHandler.preHandleBurnToken(transactionBody);
-            case TOKENDELETION -> tokenHandler.preHandleDeleteToken(transactionBody);
-            case TOKENWIPE -> tokenHandler.preHandleWipeTokenAccount(transactionBody);
-            case TOKENFREEZE -> tokenHandler.preHandleFreezeTokenAccount(transactionBody);
-            case TOKENUNFREEZE -> tokenHandler.preHandleUnfreezeTokenAccount(transactionBody);
-            case TOKENGRANTKYC -> tokenHandler.preHandleGrantKycToTokenAccount(transactionBody);
-            case TOKENREVOKEKYC -> tokenHandler.preHandleRevokeKycFromTokenAccount(transactionBody);
-            case TOKENASSOCIATE -> tokenHandler.preHandleAssociateTokens(transactionBody);
-            case TOKENDISSOCIATE -> tokenHandler.preHandleDissociateTokens(transactionBody);
-            case TOKEN_FEE_SCHEDULE_UPDATE -> tokenHandler.preHandleUpdateTokenFeeSchedule(
-                    transactionBody);
-            case TOKEN_PAUSE -> tokenHandler.preHandlePauseToken(transactionBody);
-            case TOKEN_UNPAUSE -> tokenHandler.preHandleUnpauseToken(transactionBody);
-
-            case UTIL_PRNG -> utilHandler.preHandlePrng(transactionBody);
-
+            case CONSENSUSCREATETOPIC,
```

Comment:
This is not an improvement IMO. I had a similar implementation at some point, but I chose the current approach, because it ensures, we do not forget a case. For instance the if-else block in `CryptoPreTransactionHandlerImpl.preHandle() is error-prone.